executor/prepared_test.go

```diff
@@ -47,7 +46,9 @@ func (s *testSuite) TestPrepared(c *C) {
 		plannercore.SetPreparedPlanCache(flag)
 		plannercore.PreparedPlanCacheCapacity = 100
 		plannercore.PreparedPlanCacheMemoryGuardRatio = 0.1
-		plannercore.PreparedPlanCacheMaxMemory, err = memory.MemTotal()
+		// PreparedPlanCacheMaxMemory is set to MAX_UINT64 to make sure the cache
+		// behavior would not be effected by the uncertain memory utilization.
+		plannercore.PreparedPlanCacheMaxMemory = math.MaxUint64
 		c.Assert(err, IsNil)
```

Comment:
Would you remove the line of the error checking, which is related with ``memory.MemTotal()``?